src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/body-open.html.twig

```diff
@@ -1 +1 @@
-<noscript><p><img referrerpolicy="no-referrer-when-downgrade" src="{{ analytics.content.url }}/matomo.php?idsite={{ analytics.content.siteId }}&amp;rec=1" style="border:0;" alt="" /></p></noscript>
+<noscript><p><img referrerpolicy="no-referrer-when-downgrade" src="{{ analytics.content.url|trim('/') }}/matomo.php?idsite={{ analytics.content.siteId }}&amp;rec=1" style="border:0;" alt="" /></p></noscript>
```

Comment:
why its more technical we should only rtrim urls are technical allowed to start with `//` which are "Protocol-relative URLs":

```suggestion
<noscript><p><img referrerpolicy="no-referrer-when-downgrade" src="{{ analytics.content.url|trim('/', 'right') }}/matomo.php?idsite={{ analytics.content.siteId }}&amp;rec=1" style="border:0;" alt="" /></p></noscript>
```